Multisig/UI/Settings/Passcode/EnterPasscodeViewController.swift

```diff
@@ -69,6 +69,11 @@ class EnterPasscodeViewController: PasscodeViewController {
 
         if isCorrect {
             passcodeCompletion(true, false, text)
+            do {
```

Comment:
I think this is wrong, look for didEnterEnoughSymbolsV2 method below, that method should be called instead of this in case securitycenter feature is enabled, so the question is why this method is being called instead of didEnterEnoughSymbolsV2

Reply:
we have removed didEnterEnoughSymbols couple of pr's ago and doing the security version check in didEnoughSymbols now. Having 2 functions for didEnterEnoughSymbols seemed redundant.

Reply:
I could not find `didEnterEnoughSymbolsV2` in this class or anywhere in our code.